client/src/components/Login/about/style.css

```diff
@@ -0,0 +1,31 @@
+.login__about-section{
+  background: url('./about.png');
+  flex-basis: 676px;
+}
+
+.login__about{
+  display: flex;
+  flex-direction: column;
+  width: 75%;
+  height: 100vh;
+  margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+}
+
+.login__about__heading{
```

Comment:
In BEM methodology we have only `Block`__`Element`--`Modifier` you can't use more than block
so this should just be `.login__heading {`

and the same goes for the rest